File: ejercicio2/edad.js
function boton (){
    let naci = document.getElementById("nacimiento").value;
    let actu = document.getElementById("actual").value;

    let edad = actu - naci;
    let resultado = `
        <p>La edad del cliente es: <strong>${edad}</strong></p>
    `;

    if (edad >= 18) {
        resultado += `
            <p>¡Eres mayor de edad!</p>
        `;
    }
    else {
        resultado += `
            <p>¡Eres menor de edad!</p>
        `;
    }

    document.getElementById("resultado").innerHTML = resultado;
    
}

document.getElementById("boton").addEventListener("click", boton);

File: ejercicio2/edad.py
print ("Calcularemos tu edad, y te daremos un mensajito segun ella")
naci = int(input("En que año naciste? "))
actu = int(input("En que año estamos? "))
edad = actu - naci

if edad >= 18:
    print("Eres mayor de edad")
else:
    print("Eres menor de edad")
    
print (f"tu edad es {edad} años")